### PR TITLE
chore: explist_v2 feature switch should default to on

### PR DESCRIPTION
**Pull request by ClaireNeveu**
## Description

This PR launches the new experiment list for the general userbase by changing the default state of the feature switch to "on". Individual users can choose to disable the feature by turning the feature switch off.

## Test Plan
1. Load the app
2. Go to Uncategorized and confirm it is the new experience
3. Append ?f_explist_v2=off to the url
4. Confirm you see the old experience

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
No Ticket

### CODE REVIEW

File: docs/release-notes/new-experiment-list.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+**New Features**
+
+This release includes a new experiment list page. In addition to general improved performance and stability we've added a number of new features.
```
Comment (tara-det-ai) [inline docs/release-notes/new-experiment-list.rst:5]: WebUI: In addition to improved performance and stability, we've added the following features to the experiment list page:

File: docs/release-notes/new-experiment-list.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- Metrics and hyperparameters can be selected as columns
+- The list can be filtered on any available column
+- Allow user to specify complex filters
```
Comment (tara-det-ai) [inline docs/release-notes/new-experiment-list.rst:9]: - Users can specify complex filters

**Review by tara-det-ai — APPROVED** [review]
suggested edits

**Comment by ClaireNeveu** [comment]
Thanks, @tara-det-ai

**Review by hkang1 — APPROVED** [review]
LGTM! Once it's stable do we remove this as a feature flag?

**Pull request merged**